packages/app/src/components/Restaurants/form/RestaurantForm.js

```diff
@@ -188,13 +264,13 @@ RestaurantForm.propTypes = {
     coordinates: PropTypes.shape({
       latitude: PropTypes.number.isRequired,
       longitude: PropTypes.number.isRequired
-    }),
-    showMap: PropTypes.bool.isRequired
+    })
   }),
   onSubmit: PropTypes.func,
   setRestaurant: PropTypes.func.isRequired,
   suggestTooltip: PropTypes.string,
   submitMessage: PropTypes.string,
+  autocompleteDelay: PropTypes.number
 }
 
 export default RestaurantForm
```

Comment:
This component is starting to get bloated. Might want to refactor things to smaller components and hooks later on. This is ok for now, though.